### PR TITLE
Edited answer 16 to provide fuller explanation

### PR DESCRIPTION
**Pull request by haokaiwu**
Edited answer 16 to provide fuller explanation. Also included some smaller edits elsewhere.

Mainly elaborated on the role of centralized exchanges, price volatility, stable coins, and decentralized exchanges on the ability buy and hold Ether. Left placeholders for data on the volatility of Ether vs. S&P 500 and other comparisons (may have to compile this myself from API's). 

I disagreed with the original post in that it focused on demonizing centralized exchanges. While centralized exchanges have their flaws, they're really the only way to buy Ether with USD today as a US citizen. It's also very unclear what the alternative would be until decentralized exchanges and stable coins mature as technologies. As such, it doesn't make sense to paint them as the main problem with obtaining Ether.

### CODE REVIEW

File: other/ethhub-cftc-response.md [hunk 1]
```diff
@@ -162,11 +162,23 @@ Ethereum has operated uninterrupted for three years as a decentralized blockchai
...
+
+Given the nascent markets for cryptocurrencies and longstanding laws for KYC/AML, these exchanges play a vital role in allowing investors to legally exchange Ether with USD. Even so, until their performance is tested in more market cycles, an unexpected shutdown to their operations via technical problems or business insolvency remains a significant risk to the purchase and holding of Ether.
+
+Granted, one significant caveat to the risks presented above is that investors can easily exchange Ether into other cryptocurrencies via an array of traditional exchanges, decentralized exchanges, and other services. Many of these sites, such as Shapeshift, do not require a login to exchange currencies instantaneously, making such an exchange trivial to most users. Thus, when considering the liquidity of Ether, it’s important to account for the ability to exchange any cryptocurrency (particularly other liquid ones such as Bitcoin) to USD as a secondary source of liquidity.
```
Comment (ChazSchmidt) [inline other/ethhub-cftc-response.md]: Might be useful to note that Shapeshift has previously complied with the IRS before.

File: other/ethhub-cftc-response.md [hunk 2]
```diff
@@ -162,11 +162,23 @@ Ethereum has operated uninterrupted for three years as a decentralized blockchai
...
+
+The emergence of stable coins offers an array of opportunities and risks as well. A stable coin is a cryptocurrency that has a value pegged to a real-world asset such as USD. These coins facilitate the exchange of USD and Ether by ensuring transactions are easier to track, atomic (Ether only is received if USD is sent), and cheaper/faster to send via state channels. Coinbase and Circle, two prominent firms authorized to trade USD/ETH, have launched an Ethereum-based coin pegged to USD called USDC to achieve this purpose (https://www.centre.io/pdfs/centre-whitepaper.pdf). The MakerDAO group has also released a token called DAI on Ethereum which achieves the USD peg via a decentralized system of smart contracts and collateralization (https://makerdao.com/whitepaper/).
```
Comment (ChazSchmidt) [inline other/ethhub-cftc-response.md]: - Stablecoin is probably one word in this case. 
- Might be worth mentioning Gemini and GUSD

File: other/ethhub-cftc-response.md [hunk 3]
```diff
@@ -162,11 +162,23 @@ Ethereum has operated uninterrupted for three years as a decentralized blockchai
...
+Ether can be converted to legal tender in two ways: directly via sale on an exchange or indirectly via a sale to another cryptocurrency and then to an exchange. Regarding a direct sale into fiat, Ether faces the same challenges as any other cryptocurrency: volatile price changes and immature exchanges that offer fiat-to-Ether conversion. Both factors contribute significant risk to the acquisition and holding of cryptocurrencies.
```
Comment (ChazSchmidt) [inline other/ethhub-cftc-response.md]: Does fiat currency need defined? Can we simply say fiat by itself or should it be fiat currency each time?

File: other/ethhub-cftc-response.md (hunk 1 above)
Comment (haokaiwu) [reply other/ethhub-cftc-response.md]: Good point. Will look into that. Not familiar with that situation.

File: other/ethhub-cftc-response.md (hunk 3 above)
Comment (haokaiwu) [reply other/ethhub-cftc-response.md]: I assume CFTC understands what we mean by fiat, especially in context with discussing legal tender. I think it's one of those things where you optionally say fiat currency once at the top of an answer, but only use "fiat" for the rest of the answer. As for defining it, I don't think that's necessary. The definition of fiat is clear, and if the readers don't know what fiat is, I think there are bigger problems than our lack of definition...

File: other/ethhub-cftc-response.md (hunk 2 above)
Comment (haokaiwu) [reply other/ethhub-cftc-response.md]: You're right about the stablecoin thing. I'll change it. I'll also mention GUSD.

File: other/ethhub-cftc-response.md (hunk 1 above)
Comment (haokaiwu) [reply other/ethhub-cftc-response.md]: Do you have a source on this? Couldn't find anything with a cursory look.

Comment (ChazSchmidt) [reply other/ethhub-cftc-response.md]: My mistake. I think I was remembering the IRS requesting documents from Coinbase in 2016 https://support.coinbase.com/customer/portal/articles/2924446-irs-notification

Shapeshift has moved to include KYC/AML though Shapeshift Membership

Comment (haokaiwu) [reply other/ethhub-cftc-response.md]: Nice catch. I'll change what I wrote. Unfair to use shapeshift as an example if they're doing the KYC/AML stuff

Comment (haokaiwu) [reply other/ethhub-cftc-response.md]: Latest commit edits out Shapeshift entirely and refocuses to volume of ETH/BTC transactions, which I think makes a more pertinent case.

**Pull request merged**